File: LeetCode/2427. Number of Common Factors/JavaScript/01. Number of Common Factors.js
var commonFactors = function (a, b) {
    let commonFactorValues = 0;
    const maxValue = Math.max(a, b);
    for (let i = 1; i <= maxValue; i++) {
        commonFactorValues += ((a % i == 0 && b % i == 0) ? 1 : 0);
    }
    return commonFactorValues;
};

File: LeetCode/2427. Number of Common Factors/Python/01. Number of Common Factors.py
def commonFactors(self, a, b):
    commonFactorValues = 0
    maxValue = max(a, b)
    for i in range(1, maxValue + 1):
        commonFactorValues += 1 if a % i == 0 and b % i == 0 else 0
    return commonFactorValues
